client/src/locales/en.json

```diff
@@ -87,7 +87,9 @@
   "authorization.input.password.confirm": "Confirm password",
   "authorization.input.password.invalid": "Please use min 4 and max 32 signs and don't use space characters.",
   "authorization.input.password.not-match": "Passwords do not match.",
+  "authorization.actions.sign-up.confirmation-link-sent": "In order to complete sign up process please click the link that we have just sent to {data}.",
   "authorization.actions.sign-up.user-already-exist": "User of email {data} already exist.",
+  "authorization.actions.sign-in.invalid-credentials": "The email and the password you entered didn't match our records.",
```

Comment:
maybe some better text like the google have: 'Wrong password. Try again or click Forgot password to reset it.'